src/lib/core/CHIPConfig.h

```diff
@@ -2498,6 +2498,41 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_VERBOSE_VERIFY_OR_DIE 0
 #endif
 
+/**
+ * @def CHIP_CONFIG_MAX_GROUPS_PER_FABRIC
+ *
+ * @brief Defines the number of groups supported per fabric, see Group Key Management Cluster in specification.
+ *
+ * Binds to number of GroupState entries to support per fabric
+ */
+#ifndef CHIP_CONFIG_MAX_GROUPS_PER_FABRIC
+#define CHIP_CONFIG_MAX_GROUPS_PER_FABRIC 1
+#endif
```

Comment:
Should we consider some #defines that are more realistic for this feature?  I understand these are just defaults.  But I would think a device that is going to support groupcast probably will want more than 1 of these and 1 of CHIP_CONFIG_MAX_GROUP_ENDPOINTS_PER_FABRIC.

Reply:
It's to avoid the costs on platforms that don't need them

Reply:
I was expecting the feature to be disabled, or to be enabled in a realistic configuration.

But this works too and has minimal impact.